_includes/inline-style.css

```diff
@@ -379,6 +385,10 @@ footer div a:not([rel*=license]), footer div .copyright {
 }
 
 @media screen and (min-width: 800px) {
+  footer {
+    position: sticky;
```

Comment:
using position: sticky without an inset (top/bottom/left/right) feels odd, it's got no place to stick to. what's the desired effect?